pkg/apiserver/controllers/v1/decisions.go

```diff
@@ -54,7 +66,16 @@ func (c *Controller) GetDecision(gctx *gin.Context) {
 	} else {
 		PrometheusBouncersHasEmptyDecision(gctx)
 	}
-
+	go func() {
+		bouncerKey := gctx.Request.Header.Get(c.APIKeyHeader)
+		if bouncerInfo, err := APIKeyToBouncer(bouncerKey, c.DBClient); err == nil {
+			c.BouncerPullUpdator.UpdateBouncerPullEntry(database.BouncerKey{
+				ID: bouncerInfo.ID, Name: bouncerInfo.Name,
+			})
+		} else {
+			log.Errorf("unable to fetch bouncer for key %s: %v", bouncerKey, err)
```

Comment:
Don't log the API key, it won't be really be useful for debug.

Reply:
Fixed